chat-api/v2/openapi.yaml

```diff
@@ -1069,25 +1069,83 @@ components:
         replyBack: false
     Media:
       type: object
-      description: Media to be send
+      oneOf:
+        - $ref: '#/components/schemas/AudioMedia'
+        - $ref: '#/components/schemas/ImageMedia'
+        - $ref: '#/components/schemas/DocumentMedia'
+        - $ref: '#/components/schemas/VideoMedia'
+    AudioMedia:
+      type: object
+      description: Audio to be send
       properties:
         type:
           type: string
           description: What kind of media should be send
+          enum:
+            - audio
+          example: image
+        url:
+          type: string
+          description: the url of the location where the audio file is stored
+          example: https://www.tyntec.com/themes/custom/tyntec/image/tyntec-logo-color.svg
+    ImageMedia:
+      type: object
+      description: Image to be send
+      properties:
+        type:
+          type: string
+          description: What kind of media should be send. Can only be png or jpeg
           enum:
```

Comment:
again this is a little bit overkill. enum with a single option

Reply:
see comment in the main section :)